src/globus_cli/login_manager/manager.py

```diff
@@ -66,6 +69,16 @@ class LoginManager:
             FlowsScopes.run_manage,
         ],
     }
+    # a second map is needed to track the duality of
+    #   server -> top level scope (for checking)
+    # vs
+    #   server -> top level scope with dependencies (for requesting)
+    STATIC_SCOPES_WITH_DEPENDENCIES: dict[str, tuple[list[str], list[str]]] = {
+        TIMER_RS: (
+            [TimerScopes.timer],
+            [_TIMER_SCOPE_WITH_DEPENDENCIES],
+        ),
+    }
```

Comment:
There are lots of other near-equivalent ways of phrasing this. Amongst those, I have basically no preference, as long as we allow ourselves to keep the existing scopes as-written and this is only a tweak for the timer scope declaration.

The main thing is that the login manager will need both the top level scope string and the scope string with dependencies to use in login flows.

So if you read this and think "that's a weird hack", I invite alternatives as long as they are simple to implement and fit well into the usages which required this workaround.

Reply:
I actually rewrote this to use `MutableScope`, which I realized would work out well for this purpose after starting on the sole remaining outstanding issue, mentioned above.
So I'm marking this as resolved since I think the latest version of this is fine and will be improved further by https://github.com/globus/globus-sdk-python/pull/687